application/testing/CMakeLists.txt

```diff
@@ -803,6 +803,7 @@ f3d_test(NAME TestInteractionZoomToggleOrthographicProjection DATA cow.vtp INTER
 f3d_test(NAME TestInteractionRotateCameraMinus90 DATA f3d.glb INTERACTION)
 f3d_test(NAME TestInteractionRotateCamera90 DATA f3d.glb INTERACTION)
 f3d_test(NAME TestInteractionPanWithShift DATA f3d.glb INTERACTION) #Shift;LeftMouse;MouseMovements
+f3d_test(NAME TestInteractionFileOpen DATA cow.vtp INTERACTION NO_BASELINE)
```

Comment:
It's great and much needed! But I can see that the test is failing because of timeout.

Reply:
Yeah, Im not sure there is an easy way to fix that apart some logic similar to watch test

Reply:
f3d says it will use a couple of tools. On my system I have kdialog installed so the opened file dialog is a process of kdialog. If you could add a script that waits for the test inputs to run, then checks for the kdialog process or equivalent that could check for the process then kill it and allow the rest of the tests to run. I'm not sure whether this is a good way to test it since the process name will be different depending on the system and packages.